Offroad-iOS/Offroad-iOS/Presentation/MyPage/Character/Cell/AcquiredCharactersCell.swift

```diff
@@ -46,54 +62,77 @@ class AcquiredCharactersCell: UICollectionViewCell {
     // MARK: - Setup Functions
     
     private func setupHierarchy() {
-        contentView.addSubviews(containerView, characterLabel)
+        contentView.addSubviews(containerView, characterLabel, shadowView, newBadgeView)
+        shadowView.addSubview(lockImageView)
         containerView.addSubview(imageView)
     }
     
     private func setupLayout() {
         contentView.layer.cornerRadius = 10
         contentView.clipsToBounds = true
-        contentView.backgroundColor = UIColor.home(.homeCharacterName)
         
         containerView.snp.makeConstraints { make in
-            make.height.equalTo(167)
             make.centerX.equalToSuperview()
             make.top.horizontalEdges.equalTo(contentView).inset(10)
         }
         
         imageView.snp.makeConstraints { make in
-            make.width.equalTo(75)
-            make.height.equalTo(136)
-            make.center.equalToSuperview()
+            make.width.equalTo(81)
+            make.height.equalTo(147)
+            make.centerX.centerY.equalToSuperview()
         }
         
         characterLabel.snp.makeConstraints{ make in
             make.top.equalTo(containerView.snp.bottom).offset(10)
             make.centerX.equalToSuperview()
+            make.bottom.equalToSuperview().inset(10)
+        }
+        
+        shadowView.snp.makeConstraints { make in
+            make.edges.equalTo(contentView)
+        }
+        
+        lockImageView.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.top.equalToSuperview().inset(75)
+            make.width.equalTo(33)
+            make.height.equalTo(37)
+        }
+        
+        newBadgeView.snp.makeConstraints { make in
+            make.top.trailing.equalTo(containerView).inset(8)
+            make.size.equalTo(24)
         }
     }
     
+    
     //MARK: - Func
     
-    func configureCell(imageName: String) {
-        imageView.image = UIImage(named: imageName)
+    func gainedCharacterCell(data: GainedCharacterList) {
+        imageView.fetchSvgURLToImageView(svgUrlString: data.characterThumbnailImageUrl)
+        characterLabel.text = data.characterName
+        contentView.backgroundColor = UIColor(hex: data.characterMainColorCode)
+        containerView.backgroundColor = UIColor(hex: data.characterSubColorCode)
+        
+        shadowView.isHidden = true
+        lockImageView.isHidden = true
+        
+        if data.isNewGained {
+            newBadgeView.isHidden = false
+        }
```

Comment:
P4
한 줄로 간단하게 축약할 수 있습니다!~

```suggestion
        newBadgeView.isHidden = !data.isNewGained
```